helm/prometheus-rules/templates/alerting-rules/prometheus.rules.yml

```diff
@@ -38,3 +38,15 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: PrometheusRuleFailures
+      annotations:
+        description: {{`Prometheus {{$labels.installation}}/{{$labels.cluster_id}} has failed to evaluate rule(s) {{ printf "%.0f" $value }} time(s).`}}
+        summary: Prometheus is failing rule evaluations.
+      expr: increase(prometheus_rule_evaluation_failures_total[30m]) > 0
```

Comment:
why `increase` over `rate`

Reply:
Yeah good point `rate` should be used an not `increase`.